packages/system-message-react/package.json

```
@@ -1,15 +1,15 @@
 {
-    "name": "@fremtind/jkl-alert-message-react",
+    "name": "@fremtind/jkl-system-message-react",
     "version": "11.1.19",
```

```suggestion
    "name": "@fremtind/jkl-system-message-react",
    "version": "1.0.0-alpha.0",
```
Denne vil automatisk oppdateres til `1.0.0` ved publisering